src/helpers.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+if (!function_exists('backpack_url')) {
+    /**
+     * Appends the configured backpack prefix and returns
+     * the URL using the standard Laravel helpers.
+     *
+     * @param $path
+     *
+     * @return string
+     */
+    function backpack_url($path = null)
+    {
+        $path = !$path || (substr($path, 1, 1) == '/') ? $path : '/'.$path;
+
+        return url(config('backpack.base.route_prefix', 'admin').$path);
+    }
+}
+
+if (!function_exists('backpack_avatar')) {
+    /**
+     * Returns the avatar URL of a user.
+     *
+     * @param $user
+     *
+     * @return string
+     */
+    function backpack_avatar_url($user)
+    {
+        switch (config('backpack.base.avatar_type')) {
+            case 'gravatar':
+                return Gravatar::fallback('https://placehold.it/160x160/00a65a/ffffff/&text='.Auth::user()->name[0])->get($user->email);
```

Comment:
Won't that need a `use`?

Reply:
yes I guess it will need use or we can use auth()->user() in such cases.